### ISSUE TITLE
Can't open log files

### ISSUE THREAD

**Issue by mahi97** [zerowind168/SimuroSot5-logs#1]
Hi,
Please tell how can I open .rlg files?
can I open them on Mac os or Linux?

**Comment by Winner-Nick** [comment]
You can search by GOOGLE.

**Comment by Winner-Nick** [comment]
I have found it. You must download a software called "ReliaSoft Weibull++", and then you can read it rightly!

**Comment by mahi97** [comment]
Did it work for you?

**Comment by Winner-Nick** [comment]
NO, from my region, I cannot visit some foreign websites, so I am unable to download the software.
